shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/BaseTypes.hs

```diff
@@ -454,12 +449,3 @@ instance FromCBOR Network where
     word8ToNetwork <$> fromCBOR >>= \case
       Nothing -> cborError $ DecoderErrorCustom "Network" "Unknown network id"
       Just n -> pure n
-
---------------------------------------------------------------------------------
--- Shelley Era
---------------------------------------------------------------------------------
-
-data Shelley c
```

Comment:
Particular reason for moving this? Just a feeling it should be defined at a higher level?

Reply:
I was having some issues with circular deps when parametrizing TxOut (which isnt part of this PR, but will happen eventually)